scripts/install/install.js

```diff
@@ -1,2 +1,8 @@
 var fs = require('fs')
-fs.createReadStream('scripts/install/tsqllint').pipe(fs.createWriteStream(`${process.env.APPDATA}/npm/tsqllint`))
\ No newline at end of file
+const execSync = require('child_process').execSync;
+
+// calls powershell script to install missing dependencies
+execSync(`call ${process.env.APPDATA}/npm/node_modules/tsqllint/scripts/install/install-dependencies.cmd`);
```

Comment:
good note on the missing lib, i didn't realize that. if we weren't telling people to upgrade anyway, as you mentioned, I would revisit.